web_shortcut/static/src/js/web_shortcut.js

```diff
@@ -6,6 +6,7 @@ odoo.define('web.shortcut', function (require) {
         WebClient = require('web.WebClient'),
         ViewManager = require('web.ViewManager'),
         ActionManager = require('web.ActionManager'),
+        Menu = require('web.Menu'),
```

Comment:
- When the page is refreshed on a favourite view the toggle is not 'checked'. And other cases that the toggle is not in the correct state.
- The toggle blinks every time the view changes